Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM postgres:14
```

Comment:
should we consolidate this vs the .github/Dockerfile?

Reply:
Was a plan to have a `playground` docker released to docker registry. And another docker for CI.